src/screens/Search/Result.tsx

```diff
@@ -75,13 +72,23 @@ const Result = ( {
 }: ResultProps ) => (
   <Pressable style={[ styles.root, style ]} {...props}>
     <View style={styles.header}>
-      <Typography style={styles.subText}>{toUnicode( source )}</Typography>
-      <Typography style={styles.subText}>{toUnicode( page )}</Typography>
+      <Typography style={[ styles.secondaryText, styles.headerGurmukhi ]}>
+        {toUnicode( source )}
+      </Typography>
+      <Typography style={[ styles.secondaryText, styles.headerGurmukhi ]}>
+        {toUnicode( page )}
+      </Typography>
     </View>
 
-    <Typography style={[ styles.text, styles.gurbani ]}>{gurmukhi}</Typography>
+    <Typography style={[ styles.gurbani ]}>{toUnicode( gurmukhi )}</Typography>
+
+    <Typography style={[
+      styles.translation,
+      styles.secondaryText ]}
+    >
```

Comment:
This is funky styling as the prop starts on the same like as the title but ends on a different line to the closing brace - unify by either bringing the closing brace up, or the style prop down a new line

Reply:
If I could make a controversial suggestion: It would really be nicer if we had one very good formatter.

Having worked on Python, it's a blessing to not think even in the slightest about linting issues like crossing 100 characters or being able to fit multiple lines on one after deleting a style, because the formatter does it automatically for you.

I'm not sure what is causing our setup to not work like this. Maybe it's prettier or maybe it's eslint, but something should automatically be deciding how to set up lines like the above.

For now I've manually fixed it.

Reply:
It's not controversial - we're not running prettier on the code since it only does styling and we'd like some eslint semantic-styling rules to kick in, and it's a faff to get both working well...

we should make a note to add an ESLint rule to disallow props on mismatching lines for this one I believe